lib/filesystem_utils.pm

```diff
@@ -430,7 +463,18 @@ sub validate_lsblk {
     }
 
     for my $col (keys %{$validation_test_data}) {
-        if ($validation_test_data->{$col}{value} ne $blockdev->{$col}) {
+        if ($has_mountpoints_col && $col eq 'mountpoints') {
```

Comment:
`$col` can only ever be `mountpoints` if `$has_mountpoints_col` is true.

```suggestion
        if ($col eq 'mountpoints') {
```

Reply:
that's right I had it like that and in some intermediate code made sense but not anymore,thanks.